### ISSUE TITLE
add fontInfo() method

### ISSUE THREAD

**Issue by typemytype** [typemytype/drawbot#257]
combine all `fontAscender()` `fontDescender()` into a ufo.info like object and deprecate those methods

add name table entries from https://developer.apple.com/documentation/coretext/ctfont/name_specifier_constants?language=objc

```python
info = fontInfo(fontName=None)
print(info.ascender)
```

when `fontName` is None use the current font

**Comment by justvanrossum** [comment]
... and `fontName` could also be a `fontPath`.

**Comment by typemytype** [comment]
yeah, something like `pathOrFontName` and it should try to figure out what to get where.
